grafanalib/core.py

```diff
@@ -1680,15 +1681,107 @@ def to_json_data(self):
         )
 
 
+@attr.s
+class StatValueMappings(object):
+    """
+    Generates json structure for the value mappings for the StatPanel:
+
+    :param mappingItems: List of StatValueMappingItem objects
+
+    mappings=[
+        core.StatValueMappings(
+            [
+                core.StatValueMappingItem('Offline', '0', 'red'),  # Value must a string
+                core.StatValueMappingItem('Online', '1', 'green')
+            ]
+        ),
+    ],
+    """
+    mappingItems = attr.ib(default=[], validator=instance_of(list))
+
+    def to_json_data(self):
+        ret_dict = {
+            'type': 'value',
+            'options': {
+            }
+        }
+
+        for item in self.mappingItems:
+            ret_dict['options'].update(item.to_json_data())
+
+        return ret_dict
+
+
+@attr.s
+class StatValueMappingItem(object):
+    """
+    Generates json structure for the value mapping item for the StatValueMappings class:
+
+    :param text: String that will replace input value
+    :param mapValue: Value to be replaced
+    :param color: How to color the text if mapping occurs
+    :param index: index
+    """
+
+    text = attr.ib()
+    mapValue = attr.ib(default="", validator=instance_of(str))
+    color = attr.ib(default="", validator=instance_of(str))
+    index = attr.ib(default=None)
+
+    def to_json_data(self):
+        return {
+            self.mapValue: {
+                'text': self.text,
+                'color': self.color,
+                'index': self.index
+            }
+        }
+
+
+@attr.s
+class StatRangeMappings(object):
+    """
+    Generates json structure for the range mappings for the StatPanel:
+
+    :param text: Sting that will replace input value
+    :param startValue: When using a range, the start value of the range
+    :param endValue: When using a range, the end value of the range
+    :param color: How to color the text if mapping occurs
+    :param index: index
+    """
+
+    text = attr.ib()
+    startValue = attr.ib(default=0, validator=instance_of(int))
+    endValue = attr.ib(default=0, validator=instance_of(int))
```

Comment:
As already mentioned, here `int` is used, and here: https://github.com/weaveworks/grafanalib/pull/401/files#diff-e183577edc3e65361ecd131b1d852e4b241b30f61edd379a745317aa49502339R1727 `str`